exporter/awsxrayexporter/internal/translator/aws.go

```diff
@@ -162,6 +162,14 @@ func makeAws(attributes map[string]pcommon.Value, resource pcommon.Resource) (ma
 		return filtered, nil // not AWS so return nil
 	}
 
+	// Favor Semantic Conventions for specific SQS and DynamoDB attributes.
+	if value, ok := attributes[conventions.AttributeMessagingURL]; ok {
```

Comment:
Sorry I'm having a bit of a hard time remembering here, but is there any way that this value can overwrite the QueueURL set on L152?

Reply:
Yes this is definitely possible. If the instrumentation library for the AWS SDK was setting both the X-Ray custom attribute and the Semantic Convention attribute, the latter would be favoured. This is intentional as it preserves backwards compatibility when the instrumentation library is only setting the X-Ray custom attribute, but also allows for the adoption of the sem convs in future updates to instrumentation libraries. In a nutshell, we always use the sem conv attributes if they are set.

Reply:
Got it right, had been a while since I'd gone through [these](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/aws/xray/awsxray.go)! This change is definitely the right direction going forward!

Reply:
Great, thanks for reviewing 😄